src/release/breaking-changes/page-transition-replaced-by-ZoomPageTransitionBuilder.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Page transitions replaced by ZoomPageTransitionsBuilder
+description: Using the latest page transition instead of the old one.
+---
+
+## Summary
+
+In order to keep libraries follow the latest OEM behavior,
+we've replaced the default page transition builders from
+`FadeUpwardsPageTransitionsBuilder` to
+`ZoomPageTransitionsBuilder` on all platforms excluded iOS and macOS.
+
+## Context
+
+`FadeUpwardsPageTransitionsBuilder` came with the first
+Flutter release, which defines a page transition that's
+similar to the one provided by Android O.
+
+`ZoomPageTransitionsBuilder` defines a page transition that's
+similar to the one provided by Android Q and R.
+
+According to the [Style guide for Flutter repo][], we've switched
+the default page transitions which mainly for all platforms excluded iOS and macOS.
```

Comment:
not sure what this sentence means

Reply:
It's the same as L20-23, I think it might be bad to use the same sentence.

Reply:
I don't think lines 22 and 23 here are grammatically correct English. I would suggest alternative wording but I'm not sure what they're trying to say so I'm not sure how to fix it.